src/pybind/mgr/restful/module.py

```diff
@@ -244,18 +245,43 @@ def _serve(self):
             separators=(',', ': '),
         )
 
-        cert = self.get_config_json("cert") or '/etc/ceph/ceph-mgr-restful.crt'
-        pkey = self.get_config_json("pkey") or '/etc/ceph/ceph-mgr-restful.key'
+        server_addr = self.get_config('server_addr') or '127.0.0.1'
+        server_port = int(self.get_config('server_port') or '8003')
+        self.log.info('server_addr: %s server_port: %d',
+                      server_addr, server_port)
+
+        cert_fname = None
+        pkey_fname = None
+
+        cert = self.get_config("cert")
```

Comment:
Ah, sorry, I have misread that. I thought it was self.get('config') which is a completely different thing. I suppose this should work. The only question is whether we want to do the json serialization or not but I guess for the keys, it does make sense not to serialize.